app/views/providers/details.html.erb

```diff
@@ -1,9 +1,30 @@
-<%= content_for :page_title, "About your organisation" %>
-
+<%= content_for :page_title, "#{@errors.present? ? "Error: " : ""}About your organisation" %>
 <% content_for :before_content do %>
   <%= govuk_back_link_to(provider_path(@provider.provider_code)) %>
 <% end %>
 
+<% if @errors.present? %>
```

Comment:
Do we have a partial for this?

Reply:
No, and it's slightly different between the course enrichments and provider enrichments. It's only duplicated in two locations.